### PR TITLE
Disable concurrent Jenkins builds

### LINKED ISSUE #4: Fix build race conditions in Jenkins
Notice builds 30 and 31 on Jenkins. These correspond to pushes that happened within a few seconds of each other. Build 30 started first, but finished after 31, causing the changes of build 31 to get overwritten and the website to display an old version. After noticing this, I manually ran a build to fix the problem-- but there should be a way to ensure this doesn't happen.

@jvperrin, any ideas?

### PR DESCRIPTION
**Pull request by dkess**
Fixes #4, though it's possible I messed up the syntax.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,3 +1,7 @@
+properties properties: [
+    disableConcurrentBuilds(),
+]
+
 try {
     node('slave') {
```
Comment (jvperrin) [inline Jenkinsfile]: I think I'd just put a line after this one with this:

    options { disableConcurrentBuilds() }

Seems a little cleaner, but I think what you have should work too.

**Comment by abizer** [comment]
seems like Jenkins doesn't like this

**Comment by jvperrin** [comment]
I think you need the options tag to be inside the call to `node` for it to work perhaps?

**Comment by dkess** [comment]
The one-liner was having problems, so I'm just gonna go back to the original commit.

**Comment by jvperrin** [comment]
Ok, that works too

**Pull request merged**